array_methods_lab.js

```diff
@@ -13,8 +13,9 @@ Output: [11,12,13,14,15]
 \n----------------------------------------\n\n`);
```

Comment:
Overall, good job! Code could have been cleaner. Remember to place a return within your functions.